test/test_rsb_introspection.py

```diff
@@ -55,3 +59,33 @@ def test_construction_defaults(self):
         assert isinstance(info.machine_type, str)
         assert isinstance(info.software_type, str)
         assert isinstance(info.software_version, str)
+
+
+class TestIntrospectionEvents:
+
+    @pytest.mark.usefixture('rsb_config_socket')
+    def test_introspection_events(self):
+        # The listener must be able to deserialize introspection events. But
+        # since it is created without introspection enabled, it does not force
+        # the initialization of the introspection module.
+        rsb._initialize_introspection()
+
+        # Collect events in a thread-safe way and make sure the number is right.
+        events = []
+        events_cv = threading.Condition()
+        def add_event(event):
+            with events_cv:
+                events.append(event)
+                if len(events) == 2:
```

Comment:
There's still a bit of redundancy here with all the comparisons to `2`. Maybe at least a variable that explains the meaning for that value, which is used in all places, would be good

Reply:
Done.